docs/proposals/202011_vertical_block_sharding.md

```diff
@@ -87,17 +87,49 @@ index size).
 
 ## Proposal
 
-**We propose to vertically shard all blocks which goes beyond the specified index size limit during the compaction process.**
+**We propose to vertically shard blocks when it reaches the cap limit in terms of number of series or size of index file, and then adaptively decide the number of shards depending on the users requirement**
 
-* Special metadata in meta.json indicating a shard.
+* We'll be using a special metadata in meta.json indicating a shard.
```

Comment:
Could this be another external label?

Reply:
Yes, I think `hash_number` and `shard_level` are external labels as mentioned below.